packages/rum-core/src/boot/startRum.ts

```diff
@@ -95,6 +101,22 @@ export function startRum(
   }
 }
 
+function startRumInternalMonitoring(configuration: RumConfiguration) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏This piece of code is now duplicated in both Logs and RUM. The goal is to be able to modify the RUM version to integrate it with LifeCycle in the follow-up PR, right?

Reply:
exactly, it the exact same thing for now but it will diverge to handle telemetry events